members/MNL000002.yaml

```diff
@@ -0,0 +1,56 @@
+bioguide: MNL000002
+contact_form:
+  driver: chrome
+  steps:
+    - visit: "https://www.senate.mn/members/email-form/1003"
+    - fill_in:
+        - name: First Name
+          selector: "#first_name"
+          value: $NAME_FIRST
+          required: true
+        - name: Last Name
+          selector: "#last_name"
+          value: $NAME_LAST
+          required: true
+        - name: email
+          selector: "#email"
+          value: $EMAIL
+          required: true
+        - name: street
+          selector: "#street_address_1"
+          value: $ADDRESS_STREET
+          required: true
+        - name: street2
+          selector: "#street_address_2"
+          value: $ADDRESS_STREET_2
+          required: false
+        - name: phone
+          selector: "#phone"
+          value: $PHONE_HYPHENS
+          required: false
+        - name: city
+          selector: "#city"
+          value: $ADDRESS_CITY
+          required: true
+        - name: address
```

Comment:
minor: wrong `name` here, should be zip or postal code

Reply:
good catch. Updated